addons/web/static/src/js/core/dialog.js

```diff
@@ -50,6 +50,14 @@ var Dialog = Widget.extend({
      * @param {boolean} [options.technical=true]
      *        If set to false, the modal will have the standard frontend style
      *        (use this for non-editor frontend features)
+     * @param {jQueryElement} [options.$parentNode]
```

Comment:
For the commit message of this PR: "[IMP] web: add new options to the Dialog widget" (non conjugated verb and no uppercase letter).
Also reference the task, with task-XXX.

Reply:
Done